packages/nodes-base/nodes/Schedule/ScheduleTrigger.node.ts

```diff
@@ -430,7 +431,25 @@ export class ScheduleTrigger implements INodeType {
 				Second: moment.tz(timezone).format('ss'),
 				Timezone: moment.tz(timezone).format('z Z'),
 			};
-			this.emit([this.helpers.returnJsonArray([resultData])]);
+			const weeklyExecution =
```

Comment:
If we do change the logic below to clear the initial value of `staticData[i]` then it will have the initial value of `undefined`.

Then the check below for it's definition would fail and we'd never run the correct calculations therefore set the initial value.

This also needs to be fixed.